robotidy/transformers/OrderTags.py

```diff
@@ -0,0 +1,83 @@
+from robot.api.parsing import ModelTransformer, Tags, Token
+
+
+class OrderTags(ModelTransformer):
+    """
+    Order tags inside Keywords and Test Cases.
+
+    Tags are ordered like this:
+
+        *** Test Cases ***
+        Tags Upper Lower
+            [Tags]    ba    Ab    Bb    Ca    Cb    aa
+            My Keyword
+
+        *** Keywords ***
+        My Keyword
+            [Tags]    ba    Ab    Bb    Ca    Cb    aa
+            No Operation
+
+    To:
+
+        *** Test Cases ***
+        Tags Upper Lower
+            [Tags]    aa    Ab    ba    Bb    Ca    Cb
+            My Keyword
+
+        *** Keywords ***
+        My Keyword
+            [Tags]    aa    Ab    ba    Bb    Ca    Cb
+            No Operation
+
+    Default order can be changed using following parameters:
+      - ``case_sensitive = False``
+      - ``reverse = False``
+
+    See https://robotidy.readthedocs.io/en/latest/transformers/OrderTags.html for more examples.
+    """
+    ENABLED = False
+
+    def __init__(self, case_sensitive=False, reverse=False):
+        self.key = self.get_key(case_sensitive)
+        self.reverse = reverse
+        self.separator = '    '
+
+    def visit_TestCase(self, node):
+        return self.order_Tags(node)
+
+    def visit_Keyword(self, node):
+        return self.order_Tags(node)
+
+    def order_Tags(self, node):
+        for child in node.body:
+            if getattr(child, 'type', 'invalid') in (Token.TAGS):
```

Comment:
Instead of searching for tags you can use visit_Tags. Example from robocop: https://github.com/MarketSquare/robotframework-robocop/blob/52d97f8c765566185f7aae2875a927b744d6ad61/robocop/checkers/tags.py#L46